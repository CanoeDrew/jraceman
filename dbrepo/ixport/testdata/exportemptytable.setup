CREATE table test(n int, s string not null, s2 string);
