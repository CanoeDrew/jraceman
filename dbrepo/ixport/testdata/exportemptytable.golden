
!table test
!columns "n","s","s2"
#no rows
